@ndlib/gatsby-theme-marble/src/components/App/Pages/User/UserLayout/test.js

```diff
@@ -30,8 +30,8 @@ describe('UserLayout', () => {
     expect(wrapper.find(MultiColumn).props().columns).toEqual('5')
     expect(wrapper.find(Column).length).toEqual(2)
     expect(wrapper.find(Gravatar).props().email).toEqual('me@email.web')
-    expect(wrapper.find('.identityGroup').html()).toContain('Person User')
-    expect(wrapper.find('.identityGroup').html()).toContain('person_user')
+    // expect(wrapper.find('.identityGroup').html()).toContain('Person User')
+    // expect(wrapper.find('.identityGroup').html()).toContain('person_user')
```

Comment:
I think this test should be split up rather than commenting out the lines that are failing. That way, you can use test.skip on the one that is failing, and it will show up as a skipped test in the results. Otherwise, it is easy to forget about the commented out lines and never come back to it.